client/tests/acceptance/pas-form-validation-messages-display-test.js

```diff
@@ -0,0 +1,163 @@
+import { module, test } from 'qunit';
+import {
+  visit,
+  fillIn,
+  click,
+} from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+
+module('Acceptance | pas form validation messages display', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(() => {
+    authenticateSession({
+      emailaddress1: 'me@me.com',
+    });
+  });
+
+  test('Certain fields display both Saveable and Submittable validation errors', async function(assert) {
+    this.server.create('package', {
+      project: this.server.create('project'),
+      pasForm: this.server.create('pas-form'),
+    });
+
+    // representation of input that is 1 over the maximum character limit
+    const maximum = (maxLength, singleCharacter) => singleCharacter.repeat(maxLength) + singleCharacter;
```

Comment:
Wonder if there's an easy place to store helper functions used in tests?  This seems like a useful utility function.

Reply:
Good point -- I know that we have custom test helpers, will look into if it's worth it to make this one of them!